6_visualize/in/6_outro_gif_tasks_template.mustache

```diff
@@ -29,7 +29,7 @@ targets:
       gage_sites_fun_a_{{last_task}},
       rdgs_fun_1,
       legend_fun_a_{{last_task}},
-      datetime_fun_a_{{last_task}},
+      timeline_fun_a_{{last_task}},
```

Comment:
I think we actually should just remove these from the outro entirely, as we did in Henri. In the outro, it would be frozen on the same date anyways ...